osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -321,7 +321,7 @@ protected override void OnDragEnd(DragEndEvent e)
 
         private void selectTab(int index)
         {
-            var channel = ChannelTabControl.Items.Skip(index).FirstOrDefault();
+            var channel = ChannelTabControl.Items.ElementAtOrDefault(index);
```

Comment:
will conflict with existing PR

Reply:
Whoops, forgot about that. Will change it back in a sec.